BeagleBone_black/src/motor_control/main.cpp

```diff
@@ -70,61 +92,56 @@ void Main::run()
 
 void Main::initMotors()
 {
-  if (!motors_init_) {
-    motor_data_.module_status = data::ModuleStatus::kStart;
-    motor_data_.velocity_1 = 0;
-    motor_data_.velocity_2 = 0;
-    motor_data_.velocity_3 = 0;
-    motor_data_.velocity_4 = 0;
-    motor_data_.torque_1 = 0;
-    motor_data_.torque_2 = 0;
-    motor_data_.torque_3 = 0;
-    motor_data_.torque_4 = 0;
-    data_.setMotorData(motor_data_);
+  if (!motors_init_ && !motor_failure_) {
```

Comment:
You could just `return`  if motors have been initd already or they failed rather than having everything inside an `if` but that's just a matter of taste; you don't need to change it if you don't want to.